sdk-python/agentcube/clients/client.py

```diff
@@ -14,14 +14,23 @@ class SandboxClient:
     """Pico API Server client class that encapsulates sandbox management, SSH connections, 
     and file transfer functionalities"""
     
-    def __init__(self, api_url: Optional[str] = None):
+    def __init__(
+        self, 
+        api_url: Optional[str] = None, 
+        auth_token: Optional[str] = None
+    ):
         """Initialize the client
         
         Args:
             api_url: Pico API server address. Defaults to environment variable API_URL 
                      or DEFAULT_API_URL if not provided
         """
-        self.api_url = api_url or get_env("API_URL", constants.DEFAULT_API_URL)
+        self.api_url = api_url or get_env(constants.API_URL_ENV, constants.DEFAULT_API_URL)
+        self.auth_token = auth_token or get_env("API_TOKEN", "")
```

Comment:
I am suggesting we read `/var/run/secrets/kubernetes.io/serviceaccount/token` by default